_data/snippets.yml

```diff
@@ -614,32 +614,21 @@ translation-target:
   es: Esta lección ha sido traducida al
```

Comment:
Any snippets that are no longer being used should be deleted. This way the PT team won't need to do the work of translating them needlessly.